### PR TITLE
Added description of metagenome assembly

### PR DESCRIPTION
**Pull request by hubin-keio**

### CODE REVIEW

File: src/schema/workflow_execution_activity.yaml [hunk 1]
```diff
@@ -66,6 +66,7 @@ classes:
         required: true
 
   metagenome assembly:
+    description: A workflow execution activity that dose de novo genome assembly using metagenome sequencing reads
```
Comment (wdduncan) [inline src/schema/workflow_execution_activity.yaml:69]: Should this says **does** de novo?

Comment (turbomam) [reply src/schema/workflow_execution_activity.yaml:69]: good catch

**Comment by turbomam** [comment]
Thanks, @bin!

Also, are all "metagenome assembly"s inherently de novo? They can never be guided by reference genomes?

Or are we saying that at this point in time, NMDC doesn't use any guided metagenome assemblies?

The name and description of the class should agree in a 1:1 fashion.

**Comment by turbomam** [comment]
https://www.cs.umd.edu/sites/default/files/scholarly_papers/CepedaEspinoza.pdf

**Review by turbomam — APPROVED** [review]
I moved "de novo" from the description to comments because nobody could explain NMDC's usage of de novo metagenome assemblers vs reference-guided assemblers

We c an always change that in the future

**Comment by hubin-keio** [comment]
Sorry for my late reply. The assembly is de novo in our workflow.

**Pull request merged**